crates/js-component-bindgen/src/function_bindgen.rs

```diff
@@ -0,0 +1,1195 @@
+use crate::intrinsics::Intrinsic;
```

Comment:
Function bindgen is pretty much copy-pasted from the original, but as the target for low-level bindgen.